spec/integration/inventory_spec.rb

```diff
@@ -110,36 +131,36 @@ module Hawkular::Inventory::RSpec
     end
 
     it 'Should list children of WildFly' do
-      resources = @client.list_resources_for_type(@state[:feed], 'WildFly Server')
+      resources = @client.list_resources_for_type(feed_id, 'WildFly Server')
       expect(resources.size).to be(1)
 
       wild_fly = resources[0]
 
       children = @client.list_child_resources(wild_fly)
 
-      expect(children.size).to be(21)
+      expect(children.size).to be(22)
     end
 
     it 'Should list recursive children of WildFly' do
-      resources = @client.list_resources_for_type(@state[:feed], 'WildFly Server')
+      resources = @client.list_resources_for_type(feed_id, 'WildFly Server')
       wild_fly = resources[0]
 
       children = @client.list_child_resources(wild_fly, recursive: true)
 
-      expect(children.size).to be(211)
+      expect(children.size).to be(251)
```

Comment:
yeah, the BTM module was added to Hawkular, that's why there are more resources

this is new: https://gist.github.com/Jiri-Kremser/591454a95330580249a5ae88d0a4fb4f
 (+ there are also some new JMS topics not in the gist)